src/main/webapp/assets-lti/js/utils.js

```diff
@@ -53,6 +53,26 @@ var filterOutSections = function(sectionData, mPathArray){
   return sectionData;
 };
 
+
+
```

Comment:
Maybe include a comment of why we have to do this for the benefit of future developers?